extensions/sockets/pom.xml

```diff
@@ -2,22 +2,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.mule.modules</groupId>
-        <artifactId>mule-modules</artifactId>
+        <groupId>org.mule.extensions</groupId>
+        <artifactId>mule-extensions</artifactId>
         <version>4.0-SNAPSHOT</version>
     </parent>
+
+    <groupId>org.mule.modules</groupId>
     <artifactId>mule-module-sockets</artifactId>
-    <packaging>jar</packaging>
-    <name>Sockets configuration module</name>
-    <description>A Mule module for configuring sockets.</description>
+    <packaging>mule-extension</packaging>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-spring-config</artifactId>
-            <version>${project.version}</version>
-        </dependency>
+    <name>Sockets Configuration Extension</name>
```

Comment:
this is no longer a configuration thingy. It has behaviour and actual connectivity now